code/modules/mob/living/simple_animal/hostile/morph.dm

```diff
@@ -0,0 +1,205 @@
+#define MORPH_COOLDOWN 50
+
+/mob/living/simple_animal/hostile/morph
+	name = "morph"
+	desc = "A revolting, pulsating pile of flesh."
+	speak_emote = list("gurgles")
+	emote_hear = list("gurgles")
+	icon = 'icons/mob/npc/animal.dmi'
+	icon_state = "morph"
+	icon_living = "morph"
+	icon_dead = "morph_dead"
+	speed = 2.5
+	stop_automated_movement = TRUE
+
+	response_help  = "pets"
+	response_disarm = "shoves"
+	response_harm   = "harmlessly punches"
+
+	mob_size = 15
+
+	universal_speak = TRUE
+	universal_understand = TRUE
+
+	tameable = FALSE
+
+	status_flags = CANPUSH
+	pass_flags = PASSTABLE
+
+	maxHealth = 125
+	health = 125
+
+	melee_damage_lower = 12
+	melee_damage_upper = 16
```

Comment:
i'm 99% sure there's another melee damage variable that you need to set here, but i can't remember it, harm_intent_damage?

Reply:
that affects how much damage it takes, not dishes out, iirc